.github/workflows/cifuzz.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         uses: google/oss-fuzz/infra/cifuzz/actions/run_fuzzers@master
         with:
           oss-fuzz-project-name: 'ndpi'
-          fuzz-seconds: 1200
+          fuzz-seconds: 600
```

Comment:
Does this influence fuzzing time within `oss-fuzz`?

Reply:
No, it only changes the duration of "our" fuzzing in the GitHub Actions